spk/lidarr/src/wizard/install_uifile

```diff
@@ -0,0 +1,11 @@
+[{
+    "step_title": "Starting and updating Lidarr",
+    "items": [{
+        "desc": "The first time Lidarr is started it might take a few moments for the interface to become available!<br><br>Keep Lidarr up-to-date by using Lidarr's built-in updater.<br>Navigate to System>Updates in the Lidarr UI."
+    }]
+},{
+    "step_title": "Attention! DSM Permissions",
+    "items": [{
+        "desc": "Permissions for all download-related packages are managed with the group <b>'sc-download'</b> in DSM.<br>The group 'users' is no longer used as of DSM 6.<br>Package user will not appear on most UI settings."
```

Comment:
Please report link to Permission-Management wiki page I have added to other packages (same in _fre and for upgrade wiazrds)